docs/content/integrations/dbt/reference.mdx

```diff
@@ -532,6 +532,39 @@ def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
 
 ---
 
+## Emit dbt table schema as materialization metadata <Experimental />
+
+Dagster allows you to emit table schema [materialization metadata](/concepts/assets/software-defined-assets#recording-materialization-metadata), which includes the column names and data types of your materialized dbt models, seeds, and snapshots.
+
+With this metadata, you can view documentation for all columns and not just columns described in your dbt project.
+
+To collect this metadata for your dbt project, specify the `dagster` dbt package in your `dependencies.yml`, and then run `dbt deps`.
+
+```yaml
+packages:
+  - git: "https://github.com/dagster-io/dagster.git"
+    subdirectory: "/python_modules/libraries/dagster-dbt/packages/dagster"
+    revision: {{ DAGSTER_VERSION }} # replace with the version of `dagster` you are using.
```

Comment:
Does this mean that whenever they update their Dagster version they'll need to remember to come here and update this?

Reply:
Not necessarily. If we maintain backcompat with the serialized format that we output in the macro, then users should only need to update if we add additional metadata from this macro that we want to surface in the UI.